pkgs/applications/misc/yunfaavatar/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, bash
+, makeWrapper
+, pciutils
+, x11Support ? true
+, ueberzug
+, fetchpatch
+, pkgs
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "yunfaavatar";
+  version = "unstable-2023-09-17";
```

```suggestion
  version = "0.1.0";
```
release versions preferred when possible